runelite-client/src/main/java/net/runelite/client/plugins/attackstyles/WeaponType.java

```diff
@@ -38,7 +38,7 @@
 import static net.runelite.client.plugins.attackstyles.AttackStyle.OTHER;
 import static net.runelite.client.plugins.attackstyles.AttackStyle.RANGING;
 
-enum WeaponType
+public enum WeaponType
```

Comment:
Same as above, this should be moved to a different package if it is being made public.